cmd/root.go

```diff
@@ -34,7 +34,7 @@ HPC deployments on the Google Cloud Platform.`,
 				log.Fatalf("cmd.Help function failed: %s", err)
 			}
 		},
-		Version: "v1.4.0",
+		Version: "v1.4.1",
```

Comment:
Should we roll the patch version for this? I have it in here now but can remove if we decide otherwise.

Reply:
We decided it should be a patch version update.